javascript-forms/form-events.js

```diff
@@ -0,0 +1,22 @@
+function handleFocus(event) {
+  console.log('focus event fired');
+  console.log('handleFocus event target name:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log('blur event fired');
+  console.log('handleBlur event target name:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('handleInput event target name:', event.target.name);
+  console.log('handleInput event target value:', event.target.value);
+}
+
+var $input1 = document.querySelector('#user-name');
+var $input2 = document.querySelector('#user-email');
+var $textArea = document.querySelector('#user-message');
+
+$input1.addEventListener('focus', handleFocus);
+$input2.addEventListener('blur', handleBlur);
+$textArea.addEventListener('input', handleInput);
```

Comment:
![Screen Shot 2021-02-03 at 11 21 06 AM](https://user-images.githubusercontent.com/32880187/106809405-7d2cb280-6620-11eb-80f3-9245492e3483.png)
You are missing some event handlers. You need an event handler for each of these events for every form control element.